mm2src/gossipsub/src/protocol.rs

```diff
@@ -212,6 +212,8 @@ impl Decoder for GossipsubCodec {
     type Error = io::Error;
 
     fn decode(&mut self, src: &mut BytesMut) -> Result<Option<Self::Item>, Self::Error> {
+        const MAX_TOPICS: usize = 25;
```

Comment:
With `MAX_TOPICS = 25` keep alive messages from makers who have more than 25 pairs will be ignored https://github.com/KomodoPlatform/atomicDEX-API/blob/e520117a055f920b877ae701c679b9ffabc3b2ee/mm2src/mm2_main/src/lp_ordermatch.rs#L2246-L2259
As a solution to this, we can split the above message and broadcast to 25 or less topics at a time. What do you think @ozkanonur?

Reply:
oh, i have ~ 1200 pairs per pubkey :-)

Reply:
@cipig What I meant is that they all shall be broadcasted but in multiple messages

Reply:
All trie roots for all pairs are broadcasted for each topic which is not ideal too, but having one message per pair of a pubkey might not be the best solution too since we are already getting `channel is full` errors https://github.com/KomodoPlatform/atomicDEX-API/issues/1679

Reply:
After thinking about this more, `channel is full` error might be related to this actually, @cipig has 1200 pairs and for every pair/topic the same 1200 trie roots gets sent to peers whether they are subscribed to it or not instead of only the trie roots that the peers want. Maybe one message per topic is better? What do you think @ozkanonur @Alrighttt

Reply:
I didn't take into consideration the size of the signature and pubkey in each message according to this
https://github.com/KomodoPlatform/atomicDEX-API/blob/e520117a055f920b877ae701c679b9ffabc3b2ee/mm2src/mm2_libp2p/src/lib.rs#L69-L82
One message per topic/pair would not be ideal in such case. We need to find the best value to use for the number of trie roots and topics per message taking into consideration the overhead added to each message according to this.

Reply:
We can leave the topics as it was before, since we have good validation on the message data. We don't loop over and over if the payload is invalid, if the payload is invalid, we break the loop instantly